app/views/foods/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<h4>Add Food</h4>
+<%= form_with model: @food do |form| %>
+<div class="mb-3">
+  <%= form.label :name, class:'form-label' %><br />
+  <%= form.text_field :name, class:'form-control' %>
+</div>
+
+<div class="mb-3">
+  <%= form.label :measurement_unit, class:'form-label' %><br />
+  <%= form.text_field :measurement_unit, class:'form-control' %>
+</div>
+
+<div class="mb-3">
+  <%= form.label :price, class:'form-label' %><br />
+  <%= form.number_field :price, class:'form-control' %>
+</div>
+
+<div class="actions mb-5"><%= form.submit "Add", class:'btn btn-dark' %></div>
+
+<% end %>
```

Comment:
### OPTIONAL
- [ ] Your food model has some validations. But when ı try to add food that is against validations ı am not getting any message. If users get any message about why food is not added it will be better.
**You can show error messages for other models if they have validations.**